.build/Makefile

```diff
@@ -0,0 +1,30 @@
+# set GOOARCH based on ARCH
+ifeq ($(ARCH),x86_64)
+GOARCH=amd64
+endif
+ifeq ($(ARCH),x86)
+GOARCH=386
+endif
+ifeq ($(GOARCH),)
+$(error "ARCH must be set to one of: x86, x86_64")
+endif
+
+# use "0.0.1" as default version if not supplied
+ifndef VERSION
+override VERSION=0.0.1
+endif
+
+.PHONY: package-googet
+package-googet: build pack-googet
+
+.PHONY: build
+build:
+	GOOS=windows go build -o ./bin/otelopscol_windows_$(GOARCH).exe ./cmd/otelopscol
```

Comment:
Do you plan to eventually make this command support other OSes (Linux / Mac)?

Reply:
If we go with a Makefile per build image (similar to how the existing Stackdriver agents are built), then this command will probably only support Windows. We may find, however, that we only need the one image to build the Collector, in which case we can add extra parameters to this command as needed.